### DISCUSSION TITLE
Is screenshot tests possible?

### DISCUSSION THREAD

**Discussion by eltongarbin** [ocean-ds/ocean-android#116] [Ideas]
How screenshot test is possible for android development? For the web, we have a tool to facilitate this process [chromatic](https://www.runtastic.com/blog/en/screenshot-testing-for-android/).
I want to know if there's something similar for android kotlin. It will be awesome to have this test strategy in this project, because bring us more visual quality and we can intercept a change that causes an undesirable side effect.

Here are some articles that I found:
- https://github.com/Karumi/KataScreenshotKotlin
- https://github.com/Karumi/Shot
- https://proandroiddev.com/easy-ui-and-screenshot-testing-on-android-2b138f6d1eb8
- https://blog.karumi.com/kotlin-snapshot-testing/
- https://facebook.github.io/screenshot-tests-for-android/
- https://docs.fastlane.tools/actions/screengrab/
- https://www.runtastic.com/blog/en/screenshot-testing-for-android/

**Comment by mariofelesdossantosjunior** [comment]
I believe the first step is to start with the UI tests using espresso, then just activate the screen shot.
